HopsanGUI/Dialogs/ComponentPropertiesDialog3.cpp

```diff
@@ -425,6 +434,13 @@ QDialogButtonBox *ComponentPropertiesDialog3::createOKButtonBox()
     QPushButton *pOkButton = new QPushButton(tr("&Ok"), this);
     QPushButton *pApplyButton = new QPushButton(tr("&Apply"), this);
     QDialogButtonBox *pButtonBox = new QDialogButtonBox(Qt::Horizontal, this);
+    QString filePath = mpModelObject->getAppearanceData()->getSourceCodeFile();
```

Comment:
In this case basePath is not used, maybe it should be. Or maybe getSourceCodeFile shoyld return the a solute path

Reply:
This variable is only used to check if there is a source file specified, to determine if the button should be visible or not. A bit stupid I guess, but the alternative would be to add a "hasSourceCodeFile()" function, which is not much better.